README.md

```diff
@@ -16,6 +16,18 @@ in your tests without having to directly interact with the DOM, and the chai
 plugin can be used to write readable tests with understandable output for
 mounted Vue components.
 
+> NOTE: vue-test requires the **full** version of Vue (which includes the compiler).  Make sure your build configuration for testing aliases `vue` properly.  For example (webpack 2):
+
+```javascript
+{
+  resolve: {
+    alias: {
+      'vue$': 'vue/dist/vue.esm.js'
```

Comment:
The README in Vue's dist dir has an explanation of the differences: https://github.com/vuejs/vue/blob/dev/dist/README.md

I used vue.esm.js because that's what the vue webpack template puts in your webpack config if you select the "Runtime + Compiler" option (it was recently changed to that from vue.common.js https://github.com/vuejs-templates/webpack/blob/master/template/build/webpack.base.conf.js#L25).

The reason I'm proposing this is to eliminate "duplicate vue" problems.  For example, if vue-test has 'vue/dist/vue.js', and another library specifies 'vue/dist/vue.common.js', Vue will get pulled in twice.  If everyone uses the 'vue' alias, then there will be just 1 copy (which will resolve to whatever is specified in the webpack config).

Hopefully that was clear-- let me know if you have any questions/concerns.

Thanks!